packages/medusa/src/api-v2/admin/price-lists/query-config.ts

```diff
@@ -1,53 +1,57 @@
 export enum PriceListRelations {
-  CUSTOMER_GROUPS = "customer_groups",
   PRICES = "prices",
 }
 
-export const priceListRemoteQueryFields = {
-  fields: [
-    "id",
-    "type",
-    "description",
-    "title",
-    "status",
-    "starts_at",
-    "ends_at",
-    "created_at",
-    "updated_at",
-    "deleted_at",
-  ],
-  pricesFields: [
-    "price_set_money_amounts.money_amount.id",
-    "price_set_money_amounts.money_amount.currency_code",
-    "price_set_money_amounts.money_amount.amount",
-    "price_set_money_amounts.money_amount.min_quantity",
-    "price_set_money_amounts.money_amount.max_quantity",
-    "price_set_money_amounts.money_amount.created_at",
-    "price_set_money_amounts.money_amount.deleted_at",
-    "price_set_money_amounts.money_amount.updated_at",
-    "price_set_money_amounts.price_set.variant.id",
-    "price_set_money_amounts.price_rules.value",
-    "price_set_money_amounts.price_rules.rule_type.rule_attribute",
-  ],
-  customerGroupsFields: [
-    "price_list_rules.price_list_rule_values.value",
-    "price_list_rules.rule_type.rule_attribute",
-    "price_set_money_amounts.price_rules.value",
-    "price_set_money_amounts.price_rules.rule_type.rule_attribute",
-  ],
-}
+export const adminPriceListRemoteQueryFields = [
```

Comment:
What is the difference between this one and the `defaultAdminPriceListFields` (I mean conceptually)? Maybe this should be moved inside the `listPriceLists` file instead?

Reply:
`defaultAdminPriceListFields` is the contractual fields between the consumer and the API, whereas `adminPriceListRemoteQueryFields` is the contractual fields between the API and the module. 

With the pricing module, the internal models have a lot more complicated logic in there compared to what we had in v1. I'd like to keep this here because its still a query configuration between the api and the module, until we have a better solution and a more unified approach on how to tackle this.